### DISCUSSION TITLE
ajout de fonctionnalité pour liste déroulante

### DISCUSSION THREAD

**Discussion by Arnaudvitale** [witsa/synapps#542] [Ideas]
avoir la possibilité de mettre un icône avant le texte dans les listes déroulantes.

**Comment by WITNicolas** [comment]
Réponse un peu en retard. C'est tout a fait possible.
![image](https://github.com/witsa/synapps/assets/33632664/f80a3273-e253-4187-bdd8-eadf5e288a45)

Dans le texte, il suffit de mettre : 
`<i class="icon-home"></i> <br>Accueil`

Et bien sûr de remplacer le nom de l'icone voulue
Résultat : 
![image](https://github.com/witsa/synapps/assets/33632664/202bb1af-8b0a-4df2-9aec-ed1aca9ebfcc)
